rosidl_typesupport_introspection_c/rosidl_typesupport_introspection_c-extras.cmake.in

```diff
@@ -7,6 +7,8 @@ set(rosidl_typesupport_introspection_c_LIBRARY_TYPE
   "@rosidl_typesupport_introspection_c_LIBRARY_TYPE@")
 
 find_package(ament_cmake_core QUIET REQUIRED)
+# Make sure rosidl_generator_c extension point is registered first
+find_package(rosidl_generator_c QUIET REQUIRED)
```

Comment:
Maybe remove the dependency from the manifest and the `REQUIRED` from this line?

Same for C++.

Reply:
Done